docs/dev/framework/cron.md

```diff
@@ -64,6 +64,19 @@ a Linux crontab you could use the following instructions for example:
 * * * * * php /path/to/contao/vendor/bin/contao-console contao:cron
 ```
 
+{{% notice tip %}}
+There is no HTTP request context available on the command line. However, Contao needs
+this to generate the sitemap for example. You can set the domain either in the settings of
+your website roots or you can define a default domain in your application configuration.
+See the [Symfony Routing Documentation](https://symfony.com/doc/4.4/routing.html#generating-urls-in-commands) for more details.
+{{% /notice %}}
+
+```yml
+# config/parameters.yml
+parameters:
+    router.request_context.host: 'example.org'
+    router.request_context.scheme: 'https'
+```
```

Comment:
Inside the notice please 🙃

Reply:
Ah, wasn't sure if that would work.

Reply:
Yes, we made some CSS changes to make it work. It only works within notices, if it's the last element and if there is only one paragraph.